### PR TITLE
Added myst version of x-ray tutorial.

### PR DESCRIPTION
**Pull request by melissawm**
@8bitmp3 please check - I had to do some tweaking to the format to make it work with MyST.

One problem remains: the gif is not showing properly. I'm trying to find a solution to this.

### CODE REVIEW

File: content/tutorial-x-ray-image-processing.md
Comment (8bitmp3) [inline content/tutorial-x-ray-image-processing.md:23]: @melissawm Thanks so much for driving this 👍 

Just realized something - we should mention the newly added Gaussian and Laplacian-Gaussian methods in the intro (I can open a small PR for the notebook later). Also, to improve readability for busy users, we can split the 1st paragraph into 2 shorter ones.

Suggestion:

```suggestion
This tutorial demonstrates how to read and process X-ray images with NumPy,
imageio, Matplotlib and SciPy. You will learn how to load medical images, focus
on certain parts, and visually compare them using the
[Gaussian](https://en.wikipedia.org/wiki/Gaussian_filter),
[Laplacian-Gaussian](https://en.wikipedia.org/wiki/Laplace_distribution),
[Sobel](https://en.wikipedia.org/wiki/Sobel_operator), and
[Canny](https://en.wikipedia.org/wiki/Canny_edge_detector) filters for edge
detection. 

X-ray image analysis can be part of your data analysis and
```

File: README.md
Comment (8bitmp3) [inline README.md:22]: @melissawm Just noticed that the MNIST tutorial is also 5th in the list fyi

Comment (melissawm) [reply README.md:22]: Yeah, I was waiting for that one to be merged to fix this, thanks for catching that :)

File: content/tutorial-x-ray-image-processing.md
Comment (rossbar) [reply content/tutorial-x-ray-image-processing.md:269]: Also note that the leading `- ` [turns this paragraph into a bullet](https://59-248354526-gh.circle-artifacts.com/0/site/_build/html/content/tutorial-x-ray-image-processing.html#the-laplace-filter-with-gaussian-second-derivatives) - not sure if that's intentional or not!

File: content/tutorial-x-ray-image-processing.md
Comment (rossbar) [inline content/tutorial-x-ray-image-processing.md:348]:
```suggestion
[Image normalization](https://en.wikipedia.org/wiki/Normalization_%28image_processing%29)
follows the `output_channel = 255.0 * (input_channel - min_value) / (max_value - min_value)`
[formula](http://dev.ipol.im/~nmonzon/Normalization.pdf). Because you're
using a grayscale image, you need to normalize just one channel.
```
The indentation here causes this to be interpreted [as a literal block](https://59-248354526-gh.circle-artifacts.com/0/site/_build/html/content/tutorial-x-ray-image-processing.html#the-sobel-feldman-operator-the-sobel-filter).

Note that MyST supports admonitions just like rST:
````suggestion
```{note}
[Image normalization](https://en.wikipedia.org/wiki/Normalization_%28image_processing%29)
follows the `output_channel = 255.0 * (input_channel - min_value) / (max_value - min_value)`
[formula](http://dev.ipol.im/~nmonzon/Normalization.pdf). Because you're
using a grayscale image, you need to normalize just one channel.
```
````
unfortunately since this is a MyST-specific feature, they don't render properly in `.ipynb` files, only the html.

**Review by rossbar — COMMENTED** [review]
I took the liberty of adding the missing `imageio` dependency so that the CI can run successfully (we may need to adjust the `myst_nb` config params so that failed notebook execution fails more loudly!)

Just some comments on a few MyST gotchas, mostly related to markdown being picky about dashes/numbers & spacing.

I limited my comments to the format-specific stuff and not the content of the tutorial itself - I'll take another pass once this is merged so that I can add comments with the new workflow!

File: content/tutorial-x-ray-image-processing.md
Comment (8bitmp3) [reply content/tutorial-x-ray-image-processing.md:239]: Thanks for the workaround, good idea

File: content/tutorial-x-ray-image-processing.md
Comment (8bitmp3) [reply content/tutorial-x-ray-image-processing.md:269]: The dash is intentional. 

The steps were numbered to help users follow the train of thought, since the sentence wasn't a simple one. It does look better now after the suggestion 👍 

Thanks !

File: content/tutorial-x-ray-image-processing.md
Comment (8bitmp3) [reply content/tutorial-x-ray-image-processing.md:348]: Good point

File: content/tutorial-x-ray-image-processing.md
Comment (melissawm) [reply content/tutorial-x-ray-image-processing.md:243]: Thanks, @rossbar ! This looks sufficient (and I don't think gifs will be all that common)

**Review by rossbar — APPROVED** [review]
LGTM (once the merge conflict is resolved)! Thanks @melissawm & @8bitmp3 for taking the lead on the workflow conversion here,

**Pull request merged**
